### ISSUE TITLE
staticpath does not work in cmd.exe

### ISSUE THREAD

**Issue by pomber** [meandavejustice/gh-pages-deploy#22]
When using `staticpath` in cmd.exe (in windows) files are not copied.
Probably because of the[ `cp` ](https://github.com/meandavejustice/gh-pages-deploy/blob/master/index.js#L51)command:
`if (cfg.staticpath) userCmds.push("cp -r " + cfg.staticpath + "/* .");`

**Comment by meandavejustice** [comment]
@pomber Yeah, this should be updated to just use the `fs` and `path` modules. Though it is a quick fix, not sure when I'll get around to it. Feel free to open a pr if you have the time/want it fixed soon

**Comment by qertis** [comment]
+1 System: Windows 10
